src/components/EventCardsGridContainer/index.js

```diff
@@ -35,7 +35,7 @@ const messages = defineMessages({
     defaultMessage: 'Finalize Result',
   },
   withdraw: {
-    id: 'bottomButtonText.withdraw',
+    id: 'str.withdraw',
```

Comment:
does `str.withdraw` means `withdraw` literally? if so we can use `withdraw` directly. @henryzbw what do u think

Reply:
I'm ok with `withdraw` directly instead of `str.withdraw`. But in terms of key organization, if we have the `str.` in front of it, all of the global strings will be together.